certified-connectors/Azure Communication Services SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,328 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Services SMS",
```

Comment:
@lei-msft I hope you are doing well.

I think we need to do the same work around that we did for other connectors regarding the title. Please shorten the title hear in the open source repo, and submit to ISV Studio with the short title using your last commit ID. Then we can revert the title back for you internally.